integration-tests/tests/utils/docker_utils.py

```diff
@@ -0,0 +1,73 @@
+"""Provide utility functions related to the use of Docker during integration tests."""
+
+import re
+import subprocess
+
+DOCKER_STATUS_FIELD_VALS = [
+    "created",
+    "restarting",
+    "running",
+    "removing",
+    "paused",
+    "exited",
+    "dead",
+]
+
+
+def list_prefixed_containers(docker_bin: str, prefix: str) -> list[str]:
```

Comment:
```suggestion
def get_containers_with_prefix(prefix: str) -> list[str]:
```
I feel like you can create a helper function that gets the path to the docker binary, and directly call the helper in this function.